.air.conf

```diff
@@ -25,7 +25,7 @@ delay = 1000 # ms
 
 send_interrupt = true
 
-kill_delay = 100 # ms
+kill_delay = 2000000000 # ns
```

Comment:
turns out that values is in nanoseconds, hence the change — this is timeout that `air` will use before doing a `kill -9` on the process